ci/build_python.sh

```diff
@@ -9,13 +9,22 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+package_name="cuml"
+package_dir="python"
+
+version_override=$(rapids-generate-version)
```

Comment:
The term `override` indicates something temporary / out-of-regular-order to me but in fact it's supposed to describe something that is part of standard procedure with this change.

How about just `version` ?

Reply:
Sure, I can change that.